packages/react-app/src/index.jsx

```diff
@@ -3,6 +3,7 @@ import React from "react";
 import { ThemeSwitcherProvider } from "react-css-theme-switcher";
 import { BrowserRouter } from "react-router-dom";
 import ReactDOM from "react-dom";
+import { RecoilRoot } from "recoil";
```

Comment:
Yep, is a really straightforward library for state management. It's useful since you can manage cache with it and also do some state transitions easier. 
Also it works similarly to the way hooks do, so it's pretty intuitive (for React devs heh).